emperor/core.py

```diff
@@ -109,35 +105,30 @@ def _repr_html_(self):
 
         # this provides a string representation that's independent of the
         # filesystem, it will instead retrieve them from the official website
-        output = str(self).replace('emperor_required_resources',
-                                   RESOURCES_URL)
 
+        #output = str(self).replace('emperor_required_resources',
+        #                           RESOURCES_URL)
+        output = str(self)
         # thanks to the IPython devs for helping me figure this one out
         return display(HTML(output), metadata=dict(isolated=True))
 
     def _make_emperor(self):
         """Private method to build an Emperor HTML string"""
-        pcoa_string = format_pcoa_to_js(self.ids,
-                                        self.ordination.site,
-                                        self.ordination.proportion_explained)
-
-        # we pass the mapping file headers twice so nothing is filtered out
-        mf_string = format_mapping_file_to_js(self.mapping_file_data,
-                                              self.mapping_file_headers,
-                                              self.mapping_file_headers)
-
-        # A lot of this is going to be empty because we don't really need any
-        # of it
-        footer = format_emperor_html_footer_string(False, False, False, False)
-        taxa = format_taxa_to_js([], [], [])
-        bars = format_comparison_bars_to_js([], [], 0)
-        vectors = format_vectors_to_js([], [], [], [], None)
-
-        # build the HTML string
-        output = [EMPEROR_HEADER_HTML_STRING, mf_string, pcoa_string, taxa,
-                  bars, vectors, footer]
-
-        # add the remote resources
-        _emperor = '\n'.join(output)
-
-        self._html = _emperor
+        # template = Template(TEMPLATE_STRING)
+        def listify(a):
```

Comment:
I made a few changes to the code in `examples/build.py` recently, it fixes a few minor annoyances with the object translation, I would suggest that you copy and paste from there.